antismash/common/secmet/features/feature.py

```diff
@@ -51,13 +51,16 @@ class Feature:
         subclass.
     """
     __slots__ = ["location", "notes", "type", "_qualifiers", "created_by_antismash",
-                 "_original_codon_start"]
+                 "_original_codon_start", "_pseudo"]
 
     def __init__(self, location: FeatureLocation, feature_type: str,
-                 created_by_antismash: bool = False) -> None:
+                 created_by_antismash: bool = False, pseudo: bool = False) -> None:
         assert isinstance(location, (FeatureLocation, CompoundLocation)), type(location)
         if location_bridges_origin(location):
             raise ValueError("Features that bridge the record origin cannot be directly created: %s" % location)
+        self._pseudo = bool(pseudo)
+        if self._pseudo:
+            assert not created_by_antismash, "pseudo %s features can only come from input files" % feature_type
```

Comment:
where to put this assert then? Delete or move it to to_biopython()?

Reply:
```self._pseudo = False``` is required, because otherwise it can potentially not exist to check in `is_pseudo()`.

The rest can be deleted, since it can no longer be set to `True`.